.yarn/versions/5ca86df5.yml

```diff
@@ -0,0 +1,7 @@
+releases:
+  "@radix-ui/react-context-menu": minor
+  "@radix-ui/react-dropdown-menu": minor
+  "@radix-ui/react-menu": minor
```

Comment:
I'm wondering if this will need to be `major` now because of the change in types for `defaultChecked`/`checked`/`onCheckedChange`.

@andy-hook I think technically we have to right? Because it might break people's builds otherwise (that is, if they are using `CheckboxItem` only…)

If people had built their type on top of ours it would be fine, but I highly doubt it, given it was just a boolean…

Reply:
Strictly speaking yes, though it does feel like quite the leap for such a change.

Reply:
Noted! Just to be clear then, there should be a `major` for all of the affected packages, right?

Reply:
@colepeters we're still debating I guess. I'll discuss this with @andy-hook tomorrow and we'll come to a choice and will report here.

Reply:
@colepeters We caught up and agreed major is the way to go, thanks for the contribution!